eng/linux/Generic/usr/lib/modprobe.d/99-opentabletdriver.conf

```
@@ -0,0 +1,2 @@
+install wacom /usr/bin/true
+install hid_uclogic /usr/bin/true
```

```suggestion
blacklist wacom
blacklist hid_uclogic
```

Otherwise users would need to uninstall OpenTabletDriver entirely to be able to load the vendor drivers, which may not be desirable for some setups.

---

The goal is to make OpenTabletDriver not undesirable, so I think keeping those modules unloadable is fine.

---

After further research, `modprobe -i` can circumvent `install`s - so leaving this in as-is is fine.